packages/content/package.json

```diff
@@ -54,5 +54,10 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
```

Comment:
I think it would be better to put these `devDependencies` in the root `package.json` with all the other ones used for testing.